util/attendance/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#![no_std]
+
+#[allow(unused_imports)]
+use multiversx_sc::imports::*;
+
+#[multiversx_sc::contract]
+pub trait Attendance: multiversx_sc_modules::only_admin::OnlyAdminModule {
+    #[init]
+    fn init(&self, admins: MultiValueEncoded<ManagedAddress>) {
+        self.admins().extend(admins);
```

Comment:
We need a way to extend admins after deploy

Reply:
and with that should I remove the admins register from the `init` function ?